README.md

```diff
@@ -60,6 +60,22 @@ solution = Solver().solve_Dwave_quantum_annealer(prb, token=token)
 
 **Detailed documentation and examples are available at [ReadTheDocs](https://mqt.readthedocs.io/projects/qao).**
 
+## References
+
+In case you are using MQT Bench in your work, we would be thankful if you referred to it by citing the following publication:
```

Comment:
```suggestion
In case you are using the MQT Quantum Auto Optimizer in your work, we would be thankful if you referred to it by citing the following publication:
```

Reply:
Thanks for that!